emmaa_service/api.py

```diff
@@ -2308,6 +2316,7 @@ def get(self, model):
             rv['definition'] = uniprot_client.get_function(up_id)
         else:
             rv['definition'] = None
+        _update_bioresolver(namespace, identifier, rv)
```

Comment:
Should we try getting a definition here before calling uniprot client?

Reply:
Biolookup doesn't yet provide descriptions for UniProt entries so I think this order/logic here is okay.

Reply:
Ben is correct. Adding uniprot definitions is on the todo in PyOBO